assets/src/edit-story/components/header/buttons.js

```diff
@@ -23,12 +23,12 @@ import styled from 'styled-components';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { addQueryArgs } from '@wordpress/url';
 import { Spinner } from '@wordpress/components';
```

Comment:
It appears this spinner is the only thing we use from `@wordpress/components`. That seems easily ported too,

Reply:
Yep. But I remember Sam saying that she doesn't like the one from Material Design. So maybe we have to come up with our own too.

Reply:
For the record, this was only put in while the very first version of the code was written in the PoC. It was never designed to stay in. I think 100% we need some sort of progress to show the library is loading. But we might want to flag that to Sam.

Reply:
I'll probably split that `@wordpress/components` change out of this PR, as otherwise this one is mostly ready.

Reply:
Or the spinner could be removed for now. I think components is being used for an icon too.

Reply:
Nope, that icon usage was removed a short while ago.

It's just the spinner two times.